inspirehep/ext/jinja_filters/record.py

```diff
@@ -390,3 +390,99 @@ def citation_phrase(count):
             return 'Cited 1 time'
         else:
             return 'Cited ' + str(count) + ' times'
+
+    @app.template_filter()
+    def conference_info(record):
```

Comment:
In general I think this should be merged with the logic of displaying the publication information. (while merging, the [publication information legacy code](https://github.com/inspirehep/inspire/blob/master/bibformat/format_elements/bfe_INSPIRE_publi_info.py#L28) should be double checked as I am not sure the output of the jinja macro is equal to that code)

They can both be converted into a common filter to display the publication info line.

As there are quite a few cases and the logic is not simple we need a good amount of tests for this functionality. You can have different snippets of json with all the combinations of publication information, conference info, etc and the desired output.